packages/near-membrane-base/src/connector.ts

```diff
@@ -6,10 +6,118 @@ const marshallSourceTextInStrictMode = `
+                        ) {
+                            isMarked = true;
+                        }
+                        if (!isMarked) {
+                            const evalOrigin = ReflectApply(
+                                CallSiteProtoGetEvalOrigin,
+                                callSite,
+                                []
+                            );
+                            if (
+                                typeof evalOrigin === 'string' &&
+                                ReflectApply(StringProtoIncludes, evalOrigin, [
+                                    LOCKER_IDENTIFIER_MARKER,
+                                ])
+                            ) {
+                                isMarked = true;
+                            }
+                        }
+                        // Only write a single LWS entry per consecutive LWS stacks.
+                        if (isMarked) {
+                            if (!consecutive) {
+                                consecutive = true;
+                                stackTrace += '\n    at LWS';
+                            }
+                            continue;
+                        } else {
+                            consecutive = false;
+                        }
+                        try {
+                            stackTrace += `\n    at ${ReflectApply(
+                                CallSiteProtoToString,
+                                callSite,
+                                []
+                            )}`;
+                            // eslint-disable-next-line no-empty
+                        } catch {}
+                    }
+                    return stackTrace;
+                };
+                // Make Error.prepareStackTrace non-configurable and non-writable.
+                ReflectDefineProperty(Error, 'prepareStackTrace', {
+                    // @ts-ignore: TS doesn't like __proto__ on property descriptors.
+                    __proto__: null,
+                    enumerable: true,
+                    // Error.prepareStackTrace cannot be a bound or proxy wrapped
+                    // function, so to obscure its source we wrap the call to
+                    // formatStackTrace().
+                    value: function prepareStackTrace(error: Error, callSites: NodeJS.CallSite[]) {
+                        return formatStackTrace(error, callSites);
+                    },
+                });
+                // Make Error.stackTraceLimit configurable and writable in DEV_MODE.
+                ReflectDefineProperty(Error, 'stackTraceLimit', {
+                    // @ts-ignore: TS doesn't like __proto__ on property descriptors.
+                    __proto__: null,
+                    configurable: DEV_MODE,
+                    enumerable: true,
+                    // The default stack trace limit is 10.
+                    // Increasing to 20 for wiggle room of filtered results.
+                    value: 20,
```

Comment:
@jdalton this limit on safari is 100 today. Or this code only run in v8?

Reply:
> @jdalton this limit on safari is 100 today. Or this code only run in v8?

This is only for v8.